openlibrary/templates/history/sources.html

```diff
@@ -20,22 +20,22 @@
     }
     for i in range(1, 17):
         names["SanFranPL%02d" % i] = "San Francisco Public Library"
-        
+
     def get_source_record(record_id):
         if '/' not in record_id and '_meta.mrc:' in record_id:
             record_id = 'ia:' + record_id.split('_')[0]
         item = record_id.split("/")[0]
         url = "/show-records/" + record_id
-        
+
         if item.startswith("ia:"):
             source_name = "Internet Archive"
             source_url = "//archive.org/details/" + item[3:]
         elif item.startswith("amazon:"):
             source_name = "amazon.com"
-            source_url = "http://www.amazon.com/?tag=openlibr-20"
+            source_url = "http://www.amazon.com/?tag=internetarchi-20"
```

Comment:
It seems like these should be configurable in a central location to DRY them out (and passed in through the template -- for the exact case we see here).

We may want to check that we're updating this consistently across all files:
https://github.com/internetarchive/openlibrary/search?utf8=%E2%9C%93&q=openlibr-20